src/vendor/Path.ts

```diff
@@ -38,15 +39,28 @@ export default class Path {
   }
 
   /// normalizes the path
+  /// changes slashes for Windows
   /// throws if not an absolute path
   constructor(input: string | Path) {
     if (input instanceof Path) {
       this.string = input.string
-    } else if (!input || input[0] != '/') {
-      throw new Error(`invalid absolute path: ${input}`)
-    } else {
-      this.string = sys.normalize(input)
+      return
     }
+    switch (Deno.build.os) {
+      case "linux":
+      case "darwin":
+        if (!input || input[0] != '/') {
+          throw new Error(`invalid absolute path: ${input}`)
+        } else {	    
+          this.string = sys.normalize(input)	    
+        }
+      case "windows":
+       if (!input) {
```

Comment:
This is harder. Windows absolute paths can be "[letter]:\" or "\\.*", [I believe](https://learn.microsoft.com/en-us/windows/win32/fileio/naming-a-file?redirectedfrom=MSDN)

Reply:
This is harder.  Need to figure out all the possible cases ?

Reply:
Need at least a little bit of trapping for invalid absolute paths. So, I think matching either [a-zA-Z]:/ or // will do most of it.